### PR TITLE
update kustomize owners

### PR DESCRIPTION
**Pull request by natasha41575**

### CODE REVIEW

**Comment by natasha41575** [comment]
/cc @Shell32-Natsu 

cc @KnVerey

**Review by spiffxp — COMMENTED** [review]
/ok-to-test

File: k8s.gcr.io/images/k8s-staging-kustomize/OWNERS [hunk 1]
```diff
@@ -4,12 +4,13 @@
 # https://github.com/kubernetes/org/blob/master/config/kubernetes-sigs/sig-cli/teams.yaml.
```
Comment (KnVerey) [inline k8s.gcr.io/images/k8s-staging-kustomize/OWNERS:4]: This comments states that the list is supposed to be mirroring https://github.com/kubernetes/org/blob/main/config/kubernetes-sigs/sig-cli/teams.yaml#L91-L106, which @monopole recently updated. Can you make it do that, or if that's not a good idea for some reason, change the comment?

**Review by KnVerey — COMMENTED** [review]
I see you originally were adding yourself as well, but the bot complained that you're not an org member. Are you in `kubernetes-sigs` but not `kubernetes`, and if that's the case, any reason not to get that fixed?

File: k8s.gcr.io/images/k8s-staging-kustomize/OWNERS (hunk 1 above)
Comment (natasha41575) [reply k8s.gcr.io/images/k8s-staging-kustomize/OWNERS]: I've removed the comment. I don't see a reason not to update it to mirror the other list but I'm not sure how to do that.

> Are you in kubernetes-sigs but not kubernetes, and if that's the case, any reason not to get that fixed?

This is probably the case. Is adding me to the `kubernetes` org something you are able to do?

Comment (KnVerey) [reply k8s.gcr.io/images/k8s-staging-kustomize/OWNERS]: > I don't see a reason not to update it to mirror the other list but I'm not sure how to do that.

I don't mean with automation, just copy-paste 😄 

> This is probably the case. Is adding me to the kubernetes org something you are able to do?

The process is to [open an issue](https://github.com/kubernetes/org/issues/new?assignees=&labels=area%2Fgithub-membership&template=membership.md&title=REQUEST%3A+New+membership+for+%3Cyour-GH-handle%3E) on the org repo. I can sponsor you, and I expect @monopole would be willing to too

Comment (dims) [reply k8s.gcr.io/images/k8s-staging-kustomize/OWNERS]: I am happy to sponsor as well @natasha41575

Comment (natasha41575) [reply k8s.gcr.io/images/k8s-staging-kustomize/OWNERS]: @dims @KnVerey thanks! opened the issue here: https://github.com/kubernetes/org/issues/2789

**Pull request merged**